org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/debug/firefox/FirefoxRunDebugLaunchShortcut.java

```diff
@@ -31,6 +32,15 @@ public void configureLaunchConfiguration(File file, ILaunchConfigurationWorkingC
 		wc.setAttribute(FirefoxRunDABDebugDelegate.FILE, file.getAbsolutePath());
 		wc.setAttribute(FirefoxRunDABDebugDelegate.WORKING_DIRECTORY, file.getParentFile().getAbsolutePath());
 	}
+	
+	public static boolean canLaunch(IResource resource) {
```

Comment:
This method is already defined in the parent class, no need to reimplement it.

Reply:
Ah my bad, I just realized. Fixing it now.